packages/core/src/shared/constants.ts

```diff
@@ -189,6 +189,14 @@ export const amazonQHelpUrl = 'https://aws.amazon.com/q/'
 export const amazonQVscodeMarketplace =
     'https://marketplace.visualstudio.com/items?itemName=AmazonWebServices.amazon-q-vscode'
 
+// URLs for Amazon EventBridge Scheduler
+export const eventBridgeSchedulerCreateScheduleUrl: string =
+    'https://docs.aws.amazon.com/scheduler/latest/UserGuide/schedule-types.html'
+export const eventBridgeSchedulerApiGatewayUrl: string = 
+    'https://docs.aws.amazon.com/scheduler/latest/UserGuide/managing-schedule.html'
+export const eventBridgeSchedulerRolePermissionsUrl: string =
```

Comment:
Good addition of EventBridge Scheduler related URLs. However, consider using a more descriptive naming convention for these constants. For example, `eventBridgeSchedulerCreateScheduleUrl` could be renamed to `EVENT_BRIDGE_SCHEDULER_CREATE_SCHEDULE_DOC_URL` to clearly indicate that it's a documentation URL.
```suggestion
// URLs for Amazon EventBridge Scheduler
export const EVENT_BRIDGE_SCHEDULER_CREATE_SCHEDULE_DOC_URL: string =
    ''
export const EVENT_BRIDGE_SCHEDULER_API_GATEWAY_DOC_URL: string = 
    ''
export const EVENT_BRIDGE_SCHEDULER_ROLE_PERMISSIONS_DOC_URL: string =
    ''
```